plat/kvm/arm/setup.c

```diff
@@ -212,6 +234,171 @@ static void _libkvmplat_entry2(void *arg __attribute__((unused)))
 			  (char *)cmdline, strlen(cmdline));
 }
 
+static void _init_dtb_cpu(void)
+{
+	int fdt_cpu;
+	int naddr, nsizei, cell;
+	const fdt32_t *prop;
+	uint64_t core_id, index;
+	int subnode;
+	int i;
+
+	/* Init the cpu_possible_map */
+	for (i = 0; i < MAXCPU; i++)
```

Comment:
MAXCPU is only defined when CONFIG_SMP is enabled. Build will fail if CONFIG_SMP is disabled.